package.json

```diff
@@ -89,16 +89,16 @@
   },
   "dependencies": {
     "@types/bluebird": "^3.5.30",
-    "@types/lodash": "^4.14.149",
+    "@types/lodash": "^4.14.150",
     "@types/memoizee": "^0.4.3",
-    "@types/node": "^10.17.19",
+    "@types/node": "^10.17.20",
     "abortcontroller-polyfill": "^1.4.0",
     "balena-auth": "^3.0.1",
     "balena-errors": "^4.3.2",
-    "balena-hup-action-utils": "~4.0.0",
-    "balena-pine": "^11.0.0",
+    "balena-hup-action-utils": "^4.0.1",
```

Comment:
This should still be a `~`, so that we only bump minors after the Proxy is also updated to the new version and released to prod.
It's unfortunate that this is still a thing.

Reply:
done